.github/workflows/MacOS.yml

```
@@ -192,6 +192,7 @@ jobs:
           git clone https://github.com/root-project/cling.git
           cd ./cling 
           git checkout tags/v${{ matrix.cling-version }}
+          git apply -v ../patches/llvm/cling1.2-LookupHelper.patch
```

I guess this should be
```suggestion
          git apply -v ../patches/llvm/cling1.2-LookupHelper.patch
          git apply -v ${{ github.workspace }}/CppInterOp/patches/llvm/cling1.2-LookupHelper.patch
```